.gitignore

```diff
@@ -49,6 +49,7 @@ proto/gen/*
 scripts/.java-version
 config/peer_ids.dat
 eth-reference-tests/src/test/
-eth-reference-tests/src/referenceTest/resources/eth2.0-spec-tests/
-
+eth-reference-tests/src/test-support/resources/eth2.0-spec-tests/
 
+# Deprecated - Reference Test Resources Should Be Moved from ./src/referenceTest/... to ./src/test-support/...
+eth-reference-tests/src/referenceTest/resources/eth2.0-spec-tests/
```

Comment:
I'd probably remove this from gitignore so that people see they've got a version in the old location and can delete it.

Reply:
Makes sense to me. The reason I left this was to prevent the spec test resources from being checked in while people get used to moving it over, but reviews seem like a reasonable way to prevent that as well. Will remove in my next commit.